components/Modal/ModalContentName.tsx

```diff
@@ -1,12 +1,13 @@
 import { ReactNode } from "react";
-import * as Styled from "./StyledModal";
+import * as Styled from "./Modal.styled";
 
 interface Props {
-  children: ReactNode;
+  url?: ReactNode;
+  children?: ReactNode;
 }
 
-const ModalContentName = ({ children }: Props) => {
-  return <Styled.ModalContentName>{children}</Styled.ModalContentName>;
+const ModalContentName = ({ url }: Props) => {
```

Comment:
```suggestion
const ModalContentName = ({ url, children }: PropsWithChildren<Props>) => {
```
이렇게해주면 Props에서 children 타입 안만들어주셔도 된답니다 :) 꿀팁 드리고가요~~